host/sgx/sgxquoteprovider.c

```diff
@@ -27,14 +27,41 @@ extern oe_sgx_quote_provider_t provider;
 
 void oe_quote_provider_log(sgx_ql_log_level_t level, const char* message)
 {
-    const char* level_string = level == 0 ? "ERROR" : "INFO";
-    char formatted[1024];
+    char formatted[OE_LOG_MESSAGE_LEN_MAX];
+    oe_log_level_t oe_log_level;
+    char* log_string;
 
-    snprintf(formatted, sizeof(formatted), "[%s]: %s\n", level_string, message);
+    switch ((int)level)
```

Comment:
is casting needed?

Reply:
Looks like enum is not allowed in switch statements. I get this error without type cast:
/home/ratrames/openenclave/host/sgx/sgxquoteprovider.c:34:13: error: enumeration value 'SGX_QL_LOG_NONE' not handled in switch [-Werror,-Wswitch]
    switch (level)
            ^
1 error generated.